content/docs/concepts/security/index.md

```diff
@@ -28,7 +28,7 @@ In particular, Istio security mitigates both insider and external threats agains
 {{< image width="80%" ratio="56.25%"
     link="./overview.svg"
     alt="Istio security overview."
-    caption="Istio Security Architecture"
```

Comment:
If they're going to be the same, you don't need both alt and caption to be specified. You can just specify caption.

Reply:
I see. Thanks.